cmd/mesh/main.go

```diff
@@ -39,7 +41,9 @@ func main() {
 	if err != nil {
 		log.WithField("error", err.Error()).Fatal("could not initialize app")
 	}
-	if err := app.Start(); err != nil {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
It's a little awkward that we have both `defer cancel()` and `defer app.Close` but that will go away once all the packages that `core` uses are also using the same consistent asynchronous API.